Lib/test/test_tkinter/test_geometry_managers.py

```diff
@@ -108,8 +108,8 @@ def test_pack_configure_in(self):
         a.pack_configure(in_=c)
         self.assertEqual(pack.pack_slaves(), [b, c, d])
         self.assertEqual(c.pack_slaves(), [a])
-        with self.assertRaisesRegex(TclError,
-                                    'can\'t pack %s inside itself' % (a,)):
+        with self.assertRaisesRegex(TclError, "can't pack "
+                                    '"?%s"? inside itself' % (a,)):
```

Comment:
Out PEP 8 allows an alternative for multiline calls - https://peps.python.org/pep-0008/#indentation.
And we can use triple quotes to avoid the need for implicit concatenation.
```suggestion
        with self.assertRaisesRegex(
                TclError, """can't pack "?%s"? inside itself""" % (a,)):
```